test/TestClasses.py

```diff
@@ -32,6 +32,10 @@ def implies(a: bool, b: bool):
     return (not a) or b
 
 
+def xor(a: bool, b: bool):
+    return (a and not b) or (not a and b)
```

Comment:
Why not ^?

Reply:
For some reason I thought python didn't have the xor operator...

Reply:
Removed here https://github.com/pulp-platform/pulp-nnx/pull/2/commits/772fd951234d8f0d69454d8aaec726f018f8e3a6